internal/resource/config/oauth/authserversettings/oauth_auth_server_settings_resource.go

```diff
@@ -269,7 +270,7 @@ func (r *oauthAuthServerSettingsResource) Schema(ctx context.Context, req resour
 				Description: "The roll refresh token values default policy. The default value is true.",
 				Computed:    true,
 				Optional:    true,
-				Default:     booldefault.StaticBool(true),
+				Default:     booldefault.StaticBool(false),
```

Comment:
This doesn't align with the description of the attribute. Why was this changed?

Reply:
Due to a result of my testing after a fresh spin. However, it could be based on the current config. I'll change it back and trust the description.